app/views/shared/_contributors.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= return if contributors.blank? %>
+
+<% contribs_list = [] %>
+<% contributors&.each do |contributor| %>
+  <% contribs_list << contributor %>
+<% end %>
+
+<span class="sr">Contributors: </span>
+<ul class="list-inline contributors">
```

Comment:
I don't think there's anything to do here, but I was confused when I saw the `list-inline` class here but saw the review app still showing each element on its own line. Then I saw that the `li` elements in this context pick up an `inline-block` style, which prevents them from spanning lines - which explains the behavior.

Without this class we get a bulleted list, which doesn't seem like what we want - but I guess I'm just noting that "inline" here may be a bit of a misnomer.

Reply:
I can't speak to why the style guide uses `inline-block` instead of `inline`. If I had to guess, it's probably because it results in more predictable behavior, since width/height can be set. Either way, I take your point that `list-inline-block` would be more descriptive, and a good note to recall when we rebuild the style guide.

Reply:
Agreed on all fronts here...